.github/workflows/default.yml

```diff
@@ -28,18 +28,28 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
-      - uses: actions-rs/toolchain@v1
+      - name: Install nightly `rust-fmt` 
+        uses: actions-rs/toolchain@v1
         with:
             # NOTE: Until https://github.com/actions-rs/toolchain/pull/209 is merged,
             # this should be synced with rust-toolchain.toml file.
             # After merge, this section should be removed.
             toolchain: nightly-2021-12-05
-            components: rustfmt, clippy, llvm-tools-preview
+            components: rustfmt
+            profile: minimal
+
+      - name: Install Rust toolchain 1.64 
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: 1.64
```

Comment:
Any reason for having different toolchain for clippy and llvm-tools? To have more stardnard `-C instrument-coverage`?

Reply:
There is a bug on `cargo` at `nightly-2021-12-05`, so I bumped to recent toolchain `1.64`. 
However I kept the `rust-fmt` to avoid changes on code-style. It seems that `rust-fmt` from `1.64` does not support some features inside `.rustfmt.toml`.

This is the error I got using `nightly-2021-12-05`:

```
❯ cargo b -p data-avail
Updating crates.io index
error: failed to select a version for the requirement `parity-scale-codec = "^3.0.0"`
candidate versions found which didn't match: 3.1.5, 3.1.3, 3.1.2, ...
location searched: crates.io index
required by package `kate v0.2.1 (https://github.com/maticnetwork/avail-core.git?tag=kate/v0.2.1#17c8a30a)`
... which satisfies git dependency `kate` (locked to 0.2.1) of package `fuzzing v0.1.0 (/mnt/storage/home/miguel/polygon/avail_3/fuzzing)`
```